src/resources/views/components/modals/follower.blade.php

```diff
@@ -4,33 +4,64 @@ class="fixed left-0 top-0 bg-mydark bg-opacity-50 w-full h-full justify-center i
                         <div class="text-mydark text-sm font-semibold cursor-pointer">
-                            Moon
+                            <form action="{{ route('profile.show.profile.page') }}" method="GET">
+                                <input name="userId" value="{{$follower->id}}" hidden>
+                                <button class="font-semibold text-mydark cursor-pointer">
+                                        {{ $follower->username }}
+                                </button>
+                            </form>
                         </div>
 
                         <div class="text-mydark flex font-light text-xs">
-                            33k followers
+                            {{ $user->followers->count() . ' Followers' }}
                         </div>
                     </div>
-
+                    
                     <div class="flex flex-col my-2 ml-10">
-                        <button
-                            class="flex items-center justify-center text-center text-xs font-semibold bg-mydark text-mycream hover:bg-mygray hover:text-mycream p-3 rounded-full transition-all">
-                            <x-svgs.follow-icon />
-                            Follow
-                        </button>
+                        @if(auth()->id() != $follower->id)
+                            @if(auth()->user()->followedUsers->contains($follower))
+                                <form action="{{ route('relationship.follow') }}" method="POST">
+                                    @csrf
+                                    <input name="userToFollowId" value ="{{ $user->id }}" hidden>
+                                    <button
+                                        class="flex items-center justify-center text-center text-xs font-semibold bg-mycream text-mydark hover:bg-mygray hover:text-mycream p-3 rounded-full transition-all">
+                                        <x-svgs.follow-icon />
+                                        Follow
+                                    </button>
+                                </form>
+                            @else
+                                <form action="{{ route('relationship.unfollow') }}" method="POST">
+                                    @csrf
+                                    @method('DELETE')
+                                    <input name="userToUnfollowId" value ="{{ $user->id }}" hidden>
+                                    <button
+                                        class="flex items-center justify-center text-center text-xs font-semibold bg-mycream text-mydark hover:bg-mygray hover:text-mycream p-3 rounded-full transition-all">
+                                        <x-svgs.follow-icon />
+                                        Unfollow
+                                    </button>
+                                </form>
+                            @endif 
+                        @endif 
```

Comment:
use the ternary operator for those data that you want to be dynamic. Do not disregard the DRY Principle

Reply:
Done applying DRY